plugins/smartcard/msd-smartcard-plugin.c

```diff
@@ -174,7 +191,7 @@ smartcard_inserted_cb (MsdSmartcardManager *card_monitor,
 static gboolean
 user_logged_in_with_smartcard (void)
 {
-    return g_getenv ("PKCS11_LOGIN_TOKEN_NAME") != NULL;
+        return g_getenv ("PKCS11_LOGIN_TOKEN_NAME") != NULL;
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.